docs/usage/fields.md

```diff
@@ -246,4 +281,399 @@ print(user)
 #> first_name='John' last_name='Doe'
 ```
 
-TODO: Document usage of other `Field` keyword arguments
+## Number Constraints
+
+There are fields that can be used to constrain numbers:
```

Comment:
maybe
```suggestion
There are fields metadata that can be used to constrain numbers:
```

Reply:
I guess it would be "metadata fields", but I actually prefer without the "metadata" word 🧐

Reply:
I guess it would be "metadata fields", but I actually prefer without the "metadata" word 🧐

Reply:
Yeah. metadata sounds better

Reply:
I get the feeling from `There are fields ...` that you are going to list some fields below :)
That's why I suggested field metadata

up to you, whatever you decide is fine for me as well.

Reply:
Language used going back to V1 is "arguments to constrained types" https://docs.pydantic.dev/dev-v2/usage/types/standard_types/#arguments-to-conint